### ISSUE TITLE
`consistent-boolean-name`: Document `wrappers` for refs from Vue composables

### ISSUE THREAD

**Issue by andyexeter** [sindresorhus/eslint-plugin-unicorn#3684]
Since  #3553 / #3555, `consistent-boolean-name` accepts `ref()` and `computed()` called directly on a `vue` import, and the docs mention:

> Direct Vue `ref()` calls with boolean-like values and `computed()` calls with boolean-returning functions may use boolean prefixes, such as `isBranch` or `hasDepartment`.

After reading that and hitting a false positive, the reasonable conclusion is that Vue is handled. But `isDirectVueFunctionReference` requires the callee to resolve to an `ImportSpecifier` from `'vue'` and to be named `ref` or `computed`, so a ref returned by a composable is still reported:

```js
import { useElementHover } from '@vueuse/core'

// `isHovered` starts with `is`, so it should be boolean
const isHovered = useElementHover(cardElement) // ShallowRef<boolean>
```

That covers most of VueUse and any composable you write yourself, which in a Vue codebase is a lot of the refs that want a boolean prefix.

This isn't a rule bug as `wrappers` already handles it because it resolves through the type checker rather than matching syntax:

```js
'unicorn/consistent-boolean-name': [
	'error',
	{
		wrappers: {
			Ref: 'value',
		},
	},
]
```

`Ref` alone covers the whole family: `ShallowRef` extends `Ref`, and `ComputedRef` extends `BaseComputedRef` extends `Ref`, so the `getBaseTypes` walk in `getConfiguredMemberName` finds it. Verified on a Vue 3 + VueUse codebase with `projectService: true`.

The gap is discoverability. `wrappers` is documented ~240 lines below the Vue paragraph, with a `StorageItem: 'get'` example that gives no hint it's the answer for `Ref`, and nothing links the two.

## Suggested docs change

Two lines after the existing Vue paragraph in `docs/rules/consistent-boolean-name.md`:

> Refs that come from anywhere else, such as a composable returning `ShallowRef<boolean>`, are not recognized this way. Use [`wrappers`](#wrappers) with `{Ref: 'value'}` to resolve them through the type checker instead. That covers `ShallowRef` and `ComputedRef` too, since both extend `Ref`.

I have this on a branch and would have opened it as a PR, but PR creation is restricted to collaborators.

## Aside

There's an asymmetry that may or may not be worth acting on. React gets a config-free escape hatch: `isBooleanReactReferenceVariable` accepts any `useRef` binding whose name ends in `Ref`/`Reference`. Vue has no name-based equivalent, and Vue's convention is a bare `isHovered` with no suffix. Defaulting `wrappers` to `{Ref: 'value'}` when type information is available would close the gap, but that's a behaviour change, so I've kept the suggestion above to docs only.
